www/src/components/pagination/index.js

```diff
@@ -17,8 +17,8 @@ class Pagination extends React.Component {
     const prevPageNum =
       currentPage - 1 === 1 ? `` : (currentPage - 1).toString()
     const nextPageNum = (currentPage + 1).toString()
-    const prevPageLink = isFirst ? null : `/blog/${prevPageNum}`
-    const nextPageLink = isLast ? null : `/blog/${nextPageNum}`
+    const prevPageLink = isFirst ? null : `/blog/page/${prevPageNum}`
```

Comment:
on page "2" this link would be to `/blog/page/` (which doesn't exist - we have `/blog`). I think we probably should adjust  `prevPageNum` assignment:
```diff
const prevPageNum =
-      currentPage - 1 === 1 ? `` : (currentPage - 1).toString()
+      currentPage - 1 === 1 ? `` : `page/${(currentPage - 1).toString()}`
```
and drop changes to `prevPageLink`? (Potentialy do the same for `nextPageLink` for consistency